charlib/characterizer/port.py

```
@@ -106,7 +106,26 @@ def __init__(self, noninverting_port_name: str, inverting_port_name: str, direct
         self.noninverting_port_name = noninverting_port_name
         self.inverting_port_name = inverting_port_name
 
+    def __contains__(self, item):
```

Ruff (0.15.17)

[warning] Missing return type annotation for special method `__contains__`

Add return type annotation: `bool`

(ANN204)